crates/fluvio-hub-util/src/hubaccess.rs

```diff
@@ -6,12 +6,12 @@ use surf::http::mime;
 use surf::StatusCode;
 use tracing::{debug, info};
 
+use fluvio_hub_protocol::{Result, HubUtilError};
```

Comment:
The HubUtilError naming for the error types seems like it should maybe be updated to something else. Maybe just HubError?

Reply:
Addressed on: [2e6191d](https://github.com/infinyon/fluvio/pull/2841/commits/2e6191d8a46f00d69256f3bd9a8ba91869e49c4c)